components/colorloupe/stories/template.js

```diff
@@ -4,18 +4,202 @@ import { classMap } from 'lit-html/directives/class-map.js';
 
 import '../index.css';
 import '../skin.css';
+import '../../expressvars/custom.css';
```

Comment:
You can remove this now, I've fixed this globally.

Reply:
should it just be able to access these classes(the ones imported in preview.js) or do I still need a try catch [like we do with button group](https://github.com/adobe/spectrum-css/pull/1605/files#diff-60335af04558b8caaaa1bcc366006e5ad6ddaa8b9a23efe6846248dafaaf8e90R19)?